apps/test/unit/code-studio/components/header/ProjectRemixTest.js

```diff
@@ -54,18 +54,18 @@ describe('ProjectRemix', () => {
   });
 
   it('will redirect to sign in if necessary', () => {
-    sinon.stub(window.location, 'assign');
+    sinon.stub(utils, 'navigateToHref');
```

Comment:
ChromeHeadless complains right here, because Chrome won't let you stub methods on its built-in `Location` object.